.gitmodules

```diff
@@ -14,7 +14,10 @@
 	url = ../scylla-jmx
 [submodule "scylla-tools"]
 	path = tools/java
-	url = ../scylla-tools-java
+	url = https://github.com/fruch/scylla-tools-java.git
```

Comment:
I guess you need to revert this?

Reply:
Well I think the merge who merge the tools repo, would need to remove this in sync...

Reply:
@fruch I don't understand your answer. We probably don't want this setting to your private repository, not even temporarily.

Reply:
When committing, I will change it back to ../scylla-tools-java, simultaneously merging the scylla-tools-java changes.